src/scripts/main.js

```diff
@@ -1,3 +1,59 @@
 'use strict';
 
-// write code here
+const tableBody = document.querySelector('tbody');
+const headlines = document.querySelector('thead')
+  .getElementsByTagName('th');
+const lowerHeadlines = document.querySelector('tfoot')
+  .getElementsByTagName('th');
+
+for (let i = 0; i < headlines.length; i++) {
+  headlines[i].addEventListener('click', () => {
```

Comment:
Try to use event-delegation instead of loop
https://techstacker.com/javascript-event-delegation/